src/main/resources/db/migration/V1_52__physical_attributes_migration.sql

```diff
@@ -0,0 +1,13 @@
+drop table prison_person_migration_mapping;
+
+create table prison_person_migration_mapping (
+  nomis_prisoner_number          varchar(7) not null,
+  migration_type                 varchar(20)  not null,
+  dps_ids                        varchar(200) not null,
```

Comment:
is this going to be big enough? How many ids are you expecting in here? How are the ids generated?